test/extended/prometheus/prometheus.go

```diff
@@ -213,7 +213,7 @@ var _ = g.Describe("[Feature:Prometheus][Conformance] Prometheus", func() {
 			tests := map[string][]metricTest{
 				// should be checking there is no more than 1 alerts firing.
 				// Checking for specific alert is done in "should have a Watchdog alert in firing state".
-				`ALERTS{alertstate="firing"}`: {metricTest{greaterThanEqual: false, value: 2}},
+				`sum(ALERTS{alertstate="firing"}) by (alertname)`: {metricTest{greaterThanEqual: false, value: 1}},
```

Comment:
Should this maybe rather be a count? The value of the time-series is the result of its expression at alert evaluation time, so this may very well be 0, legitimately even if an alert is firing.